server/database/seeders/20190530205345-demo-users.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+'use strict';
+const bcrypt = require('bcryptjs');
+const salt = bcrypt.genSaltSync(10);
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('Users', [{
+      firstName: 'Jamie',
+      lastName: 'Foxx',
+      email: 'jamiefoxx@gmail.com',
+      password: bcrypt.hashSync('jamiefoxx', salt),
```

Comment:
you can just pass 10 in place of salt, i.e `bcrypt.hashSync('jamiefoxx', 10)`, then you can get rid of line 4 as well

Reply:
Okay, noted

Reply:
Done